src/cmds/init.py

```diff
@@ -310,11 +310,14 @@ def chia_init(root_path: Path):
             f"Please unset the environment variable and run chia init again\n"
             f"or manually migrate config.yaml"
         )
-        if root_path.exists():
-            print(f"Not migrating because {root_path} already exists")
-            return
 
     print(f"Chia directory {root_path}")
+    if root_path.is_dir() and Path(root_path / "config" / "config.yaml").exists():
```

Comment:
This is actually the way we were doing it before, which we changed for rc9 because we had to migrate testnet dir to testnet dir.